src/coffee-script.coffee

```diff
@@ -146,13 +163,13 @@ exports.run = (code, options = {}) ->
 
   # Set the filename.
   mainModule.filename = process.argv[1] =
-    if options.filename then fs.realpathSync(options.filename) else '.'
+    if options.filename then fs.realpathSync(options.filename) else '<anonymous>'
```

Comment:
I can't see any obvious issue, but on the other hand I have no idea if there was anything special about the `.` before either.